server/routes/order.js

```diff
@@ -0,0 +1,61 @@
+const { Chef,Customer,Order, User } = require("../models/index");
+const express = require("express");
+const router = express.Router();
+const _ = require("lodash");
+const Joi = require("joi");
+const createError = require("http-errors");
+const { decodeToken, userIsAuthorized } = require("../middleware/auth");
+const fileUploadService = require("../services/fileUploader");
+
+/**
+ * GET chef's or customer's orders
+ */
+router.get("/:user_id", async (req, res, next) => {
+
+    const user = await User.findById(req.params.user_id);
+
+  res.status(200).send(user.orders);
+});
+
+/**
+ * POST to /order/:chef_id/:customer_id to create new order 
+ *  and push order_id to chef's and customer's orders
+ */
+router.post("/:chef_id/:customer_id", async (req, res, next) => {
+    const { error } = validateOrder(req.body);
+    if (error) return next(createError(400, error.details[0].message));
+
+    const order = await Order.create(req.body);
+    const chef = await Chef.findById(req.params.chef_id);
```

Comment:
As an alternative, we could add a pre('save') hook to Order model similar to the hook that exists on the Dish model for adding new dish to its chef's document